src/xSTIR/pSTIR/tests/test_algebra.py

```diff
@@ -63,16 +77,32 @@ def test_division_by_datacontainer_zero(self):
 
 
 class TestSTIRAcquisitionDataAlgebraMemory(unittest.TestCase, TestDataContainerAlgebra):
+
     def setUp(self):
         pet.AcquisitionData.set_storage_scheme('file')
-        path = os.path.join(
-            examples_data_path('PET'), 'thorax_single_slice', 'template_sinogram.hs')
+        sirf_path = os.environ.get('SIRF_PATH')
+        data_path = os.path.join(sirf_path, 'tests', 'data.hs')
+        if os.path.exists(data_path):
+            print('reading data from %s' % data_path)
+            data = pet.AcquisitionData(data_path)
+            self.image1 = data.get_uniform_copy(0)
+            self.image2 = data.get_uniform_copy(0)
+            return
+        path = os.path.join(examples_data_path('PET'),
```

Comment:
not used?

Reply:
> looks good. I didn't check current run-time.
> 
> Where does tests/data.hs sit? Is it checked in ? (I didn't check)

testing time is now 14 sec

`data.hs` and `data.s` are created, if not present, in `SIRF/tests` by the first test - do not know how to delete them after all tests finished (CMake can, I presume?)

> not used?

sorry, do not get what you refer to